js/hubs/components/ReleasePurchase.js

```diff
@@ -204,45 +202,46 @@ const ReleasePurchase = (props) => {
         </Typography>
       )}
       <HubsModal releasePubkey={releasePubkey} metadata={metadata} />
-      <form
-        onSubmit={handleSubmit}
-        style={{
-          textAlign: 'left',
-          marginBottom: '8px',
-          marginTop: { md: '0px', lg: '8px' },
-        }}
-      >
-        <BuyButton
-          variant="outlined"
-          type="submit"
-          soldOut={release.remainingSupply === 0}
-          disabled={release.remainingSupply === 0 ? true : false}
+      <Box display="flex" flexDirection="row" justifyContent="space-between">
+        <Box
+          sx={{
+            width: '50%',
+            paddingRight: '4px',
+          }}
         >
-          <BuyButtonTypography
+          <BuyButton
+            fullWidth
+            variant="outlined"
             soldOut={release.remainingSupply === 0}
-            variant="body2"
-            align="left"
+            disabled={release.remainingSupply === 0 ? true : false}
```

Comment:
we can refactor this to release.remainingsupply === 0 and return the same boolean value